### PR TITLE
[NO-TICKET] Content feedback for color migration page

### PR DESCRIPTION
**Pull request by line47**
## Summary

- Update content based on designer feedback 
- Add links to theme colors to help with context in the what does it mean callouts
- Added a new section for components impacted as a way to give details on where the system components change

## How to test

1. run the doc site with `yarn start` and visit the Standardizing theme colors page under the migration guides left nav. 

## Checklist

- [ ] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title`

### If this is a change to code:

- [ ] Created or updated unit tests to cover the logic added
- [ ] If necessary, updated unit-test snapshots (`yarn test:unit:update`) and browser-test snapshots (`yarn test:browser:update`)

### If this is a change to documentation:

- [ ] Checked for spelling and grammatical errors

### CODE REVIEW

File: packages/docs/content/migration-guides/standardize-theme-colors.mdx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Theme colors normalization update
+title: Standardizing theme colors
```
Comment (pwolfert) [inline packages/docs/content/migration-guides/standardize-theme-colors.mdx:2]: The reason I included the word "update" was to signal that we're talking about a specific update, but I struggled to make this unique. What if we said something like "2023 theme-color standardization" or "v7 theme-color standardization" or "Standardizing theme colors in v7"?

File: packages/design-system-tokens/src/themes/core.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const themeColors: ColorTokens = {
   'error-dark':                 color['rose-600'],
   'error-darker':               color['rose-700'],
   'error-darkest':              color['rose-800'],
-  'error-light':                color['rose-200'],
+  'error-light':                color['rose-300'],
```
Comment (pwolfert) [inline packages/design-system-tokens/src/themes/core.ts:34]: I think this kind of further standardization should happen in a separate pull request into `main`.

Also if we're updating these colors, should we update each of the individual status color sections to also talk about the color shift in the semantic versions? Like perhaps the `## Red` section becomes `## Error / Red` and below the table that shows red disappearing, we have a table that shows `error-light` changing from `rose-200` to `rose-300`

File: packages/docs/content/migration-guides/standardize-theme-colors.mdx [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 order: 60
-intro: We've updated the theme color variables to exclude several families of non-semantic color variables and normalize color variables across themes.
+intro: We've updated theme colors by removing non-semantic colors red, gold, green and coolblue and standarizing colors across themes.
```
Comment (pwolfert) [inline packages/docs/content/migration-guides/standardize-theme-colors.mdx:4]: 👍

**Review by pwolfert — COMMENTED** [review]
I like the edits 👍 

I do want to separate actual color changes from the documentation, though. Since our other ticket for color normalization has been closed, we should probably create another ticket to track these other normalizations under

**Comment by pwolfert** [comment]
I created [a commit that essential undoes the color changes](https://github.com/CMSgov/design-system/pull/2393/commits/0dddff0eb80a67e8f563cdbdb00644a31343ead1) which we can apply in reverse to another branch once we have a ticket for the additional color changes.

**Comment by line47** [comment]
Thanks @pwolfert !

File: packages/design-system-tokens/src/themes/core.ts (hunk 2 above)
Comment (line47) [reply packages/design-system-tokens/src/themes/core.ts]: Cool, PR here! https://github.com/CMSgov/design-system/pull/2394

**Comment by line47** [comment]
PR into main on color ramp updates https://github.com/CMSgov/design-system/pull/2394

File: packages/docs/content/migration-guides/standardize-theme-colors.mdx (hunk 1 above)
Comment (pwolfert) [reply packages/docs/content/migration-guides/standardize-theme-colors.mdx:2]: What do you think about this, @line47?

Comment (line47) [reply packages/docs/content/migration-guides/standardize-theme-colors.mdx:2]: That sounds great to me and I agree with the clarification! I like "Standardizing theme colors in v7"

Comment (pwolfert) [inline packages/docs/content/migration-guides/standardize-theme-colors.mdx:2]:
```suggestion
title: Standardizing theme colors in v7
```

**Review by pwolfert — APPROVED** [review]

**Pull request merged**